### PR TITLE
Do not check for end of line after CRTF version number

### PR DESCRIPTION
**Pull request by keflavich**
Issue #239 reported a CRTF file with text after the CRTF version number on the first line, which our parser failed to read because it searched for and end-of-line after the version.  I don't believe the end-of-line is a necessary part of the CRTF spec.

### CODE REVIEW

**Comment by keflavich** [comment]
Do not merge this until we add tests.

**Comment by bsipocz** [comment]
This PR is on a branch in the main repo, TODO: delete branch once merged.

**Review by larrybradley — APPROVED** [review]
Thanks, @keflavich.  I've added a regression test and changelog entry.

This PR fixes one of the issues reported in #239.  I've discovered another issue and will submit a fix in a separate PR.

**Pull request merged**
